.github/workflows/ci.yml

```diff
@@ -11,15 +11,24 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Install uv
-        uses: astral-sh/setup-uv@v3
-
       - name: Set up Python
-        run: uv python install 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
 
-      - name: Install the dependencies
-        run: uv sync 
+      - name: Create and activate virtual environment
+        run: |
+          python -m venv .venv
+          echo "VIRTUAL_ENV=${{ github.workspace }}/.venv" >> $GITHUB_ENV
```

Comment:
I do not think it makes much sense to create a virtul env. You install python from scratch, no point in doing this.

Reply:
@mvechtomova what should I do then if I'm using venv?